### ISSUE TITLE
ChallengeActivity NullPointerException crash

### ISSUE THREAD

**Issue by ilians** [Adyen/adyen-android#404]
**Bug description**
When testing the drop in implementation the ChallengeActivity crashes on launch due to a null pointer when trying to animate a view.

Using the Mastercard test card:
https://docs.adyen.com/development-resources/test-cards/test-card-numbers#test-3d-secure-2-authentication

**Steps:**
1. Launch drop in
2. Click on payment method credit card
3. Fill in test credit card details (5454 5454 5454 5454 | 03/2030 | 737)
4. Enter Name on card: "My name"
5. Click the pay button
5. Challenge activity is launched 
6. Challenge activity crashes shortly after and the screen is closed

**Version:**
 - 4.0.0-beta03
 - 3.8.2

**Smartphone:**
 - Device: OnePlus 6
 - OS: OxygenOS 10.3.11
 - Version :Android 10

**App Theme**
`<style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">`

**Drop in config**
```
    private fun createConfig(context: Context): DropInConfiguration {
        return DropInConfiguration.Builder(context, AdyenDropInService::class.java, BuildConfig.ADYEN_CLIENT_KEY)
            .setEnvironment(environment)
            .setShopperLocale(locale)
            .addCardConfiguration(createCardConfig(locale))
            .build()
    }

    private fun createCardConfig(locale: Locale): CardConfiguration {
        return CardConfiguration.Builder(locale, environment, BuildConfig.ADYEN_CLIENT_KEY)
            .setHolderNameRequired(true)
            .build()
    }
```

**Stacktrace**
```
    java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setAlpha(float)' on a null object reference
        at android.view.View$3.setValue(View.java:27146)
        at android.view.View$3.setValue(View.java:27143)
        at android.animation.PropertyValuesHolder$FloatPropertyValuesHolder.setAnimatedValue(PropertyValuesHolder.java:1379)
        at android.animation.ObjectAnimator.animateValue(ObjectAnimator.java:990)
        at android.animation.ValueAnimator.setCurrentFraction(ValueAnimator.java:684)
        at android.animation.ValueAnimator.setCurrentPlayTime(ValueAnimator.java:647)
        at android.animation.ValueAnimator.start(ValueAnimator.java:1079)
        at android.animation.ValueAnimator.start(ValueAnimator.java:1098)
        at android.animation.ObjectAnimator.start(ObjectAnimator.java:852)
        at android.animation.ValueAnimator.startWithoutPulsing(ValueAnimator.java:1091)
        at android.animation.AnimatorSet.handleAnimationEvents(AnimatorSet.java:1149)
        at android.animation.AnimatorSet.startAnimation(AnimatorSet.java:1234)
        at android.animation.AnimatorSet.start(AnimatorSet.java:729)
        at android.animation.AnimatorSet.start(AnimatorSet.java:684)
        at com.adyen.threeds2.internal.ui.activity.a.a(:58)
        at com.adyen.threeds2.internal.ui.activity.a.a(:49)
        at com.adyen.threeds2.internal.ui.activity.a.c(:10)
        at com.adyen.threeds2.internal.ui.activity.a.b(:38)
        at com.adyen.threeds2.internal.ui.activity.ChallengeActivity.a(:25)
        at com.adyen.threeds2.internal.ui.activity.ChallengeActivity.onNewIntent(:3)
        at android.app.Activity.performNewIntent(Activity.java:7829)
        at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1473)
        at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1491)
        at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:3576)
        at android.app.ActivityThread.handleNewIntent(ActivityThread.java:3588)
        at android.app.servertransaction.NewIntentItem.execute(NewIntentItem.java:53)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2109)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7682)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

**Comment by caiofaustino** [comment]
Hi there,

Sorry but I couldn't reproduce your issue here. Does it happen 100% of the time?

**Comment by ilians** [comment]
Yes, 100%

**Comment by ilians** [comment]
https://user-images.githubusercontent.com/921529/120336167-5fcbd480-c2f2-11eb-9f6c-289885bd6719.mp4
Even though it doesn't really show much, I've attached a short video of the exception flow.

**Comment by caiofaustino** [comment]
hey, sorry for the delay, I can't really think of much to help debug here. Would it be possible for you to make a separate test application just with the parts needed to implement the Drop-in and see if you can still reproduce this? I think it might be something related to the combination of the SDK and your project since we haven't seen this reported before and we also can't reproduce it.

**Comment by ilians** [comment]
After a long debugging session I tracked the issue back to the debug menu library we use in our app.
This lib was trying some funny business with the ChallengeActivity which probably caused an error in the view inflation / findViewById leading to the null pointer. 

https://github.com/pandulapeter/beagle#crash-when-opening-a-third-party-activity

> Beagle works by adding a Fragment on top of every Activity's layout.... etc

After disabling the library for third party activities the screen loaded and I was presented the challenge screen 🎉 .
Thanks for the support, not much of a bug after all.. issue has been resolved.
